### DISCUSSION TITLE
Can I switch to the on the left/upper/blow/right window using <mod>+hjkl?

### DISCUSSION THREAD

**Discussion by haphamdev** [ianyh/Amethyst#1683] [Q&A]
Question 1:
Clockwise and counter-clockwise rotation does not fit me well. 
I'm a tmux user where I switch between tmux panes using `Ctrl+hjkl`.
Is it possible on Amethyst?

Question 2:
*An advanced question **if question 1 has a solution**.*
Does Amethyst support a Command line interface?
I want to share the same key -bindings for switching tmux panes and switching Amethyst windows.

**Comment by ianyh** [comment]
This is not currently possible. With how things are structured it would be a bit awkward to implement, but it is certainly possible.

**Comment by spacetime-labs** [comment]
Hoping we get this too, would love to manouver with [mod]+hjkl.
